packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.tsx

```diff
@@ -89,7 +89,7 @@ export class GroupedList extends BaseComponent<IGroupedListProps, IGroupedListSt
         className={ css('ms-GroupedList', styles.root, className) }
         data-automationid='GroupedList'
         data-is-scrollable='false'
-        role='grid'
+        role='group'
```

Comment:
Rowgroup requires its child element to have atleast one role="row".
Required Owned Elements:	row
Internally we have designed groupedList as List with ListItem and not as row